microsoftgraph/microsoft-graph-explorer-v4
# Feature: Unconsent permissions

## Overview

Closes #1844 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Checkout to this branch

### Test permissions tablist UI

* Open Modify Permissions tab and observe fixes as pointed out on the issue

### Test revoking of permissions

* Try revoking without consenting to the required permissions that enable revoking i.e. `DelegatedPermissionGrant.ReadWrite.All` and `Directory.Read.All` and observe behavior & status message

_Ensure that you have consented to the required permissions to enable consent before trying the following tests_
* Try revoking a default permission e.g. `User.Read` and observe behavior & status message.
* Try revoking to any other permission and observe behavior.
